src/main/fc/fc_core.c

```diff
@@ -513,6 +513,73 @@ void tryArm(void)
     }
 }
 
+static void handlePIDAntiWindup(throttleStatus_e throttleStatus)
+{
+    /* In airmode Iterm should be prevented to grow when Low thottle and Roll + Pitch Centered.
+       This is needed to prevent Iterm winding on the ground, but keep full stabilisation on 0 throttle while in air
+       Low Throttle + roll and Pitch centered is assuming the copter is on the ground. Done to prevent complex air/ground detections */
+
+    static bool antiWindupWasDeactivatedOnce = false;
```

Comment:
This can be handled by runtime_config.h STATE just like `ANTI_WINDUP` and other states used here